apps/freelance/src/pages/ProfileQuestions1/hooks.ts

```diff
@@ -0,0 +1,19 @@
+import { profileQ1Payload } from 'redux/services/types/profileQuestions1.types';
+
+import { IProfileQuestions1 } from './model';
+
+export const onFinishLogic = (values: IProfileQuestions1): profileQ1Payload => {
+  const jsonedEdu = JSON.parse(JSON.stringify(values.education));
```

Comment:
look weared, try to find solution without do difficult transformation

Reply:
fixed (added Moment library and fixed typescript)